components/eam/src/chemistry/yaml/modal_aero_amicphys/f90_yaml/gas_aer_uptkrates_1box1gas_beg_yml.f90

```diff
@@ -0,0 +1,107 @@
+#ifdef YAML_GAS_UPTKRATE
+  !-----------------------------------------------------------------------------------------
+  !"lchnk" is needed for the following code to work,
+  ! temporarily pass it along from upper level subroutines
+  ! as y_lchnk and uncomment the following code:
+  ! integer, intent(in) :: y_lchnk
+  !-----------------------------------------------------------------------------------------
+
+  !-----------------------------------------------------------------------------------------
+  ! **OR** if this subroutine is called in a nested loop of columns and levels,
+  ! we also might need column index (y_i or icol) and level
+  ! index (y_k or klev) to be passed to this routine and
+  ! uncomment the following code:
+  integer, intent(in) :: y_i, y_k, y_lchnk
+  !-----------------------------------------------------------------------------------------
+
+  !-----------------------------------------------------------------------------------------
+  ! This is used when multiple sets of yaml output is needed
+  !to cover different options (e.g., true and false)
+  ! character(len=200) :: ext_str
+  !-----------------------------------------------------------------------------------------
+
+  type(yaml_vars) :: yaml
+  integer  :: unit_input, unit_output, y_nstep
+
+  ! some subroutines are called multiple times in one timestep, record the number of calls
+  integer,save :: n_calls=0
+
+
+  !populate YAML structure
+  !(**remove yaml%lev_print, nstep_print, col_print if generating data for a dependent subroutines**)
+  yaml%lev_print = 68       !level
```

Comment:
@mingxuanwupnnl Can we try to generate files with the same latitude, longitude, time step, and level that @brigaudet used in his [PR # 183](https://github.com/eagles-project/e3sm_mam4_refactor/pull/183/files#diff-25722bfcf233098c77874783b2d1f5dfa09a94af030959beead56fb20103ff92)?

Reply:
@odiazib I added new files to be consistent with @brigaudet , I still keep the previous files which are consistent with pcarbon aging subroutine.